pkg/cluster/oidc.go

```diff
@@ -18,7 +20,7 @@ func oidcEntry() *ConfigManagerEntry {
 			func(c *Config) error {
```

Comment:
I like the direction you are taking with this
Maybe we should consider changing the signature of these validators to return an slice of errors in the long term. Wdyt?

Reply:
That makes sense! Something like changing [this line](https://github.com/aws/eks-anywhere/blob/main/pkg/cluster/manager_entry.go#L11) to return a slice of errors? Then refactoring the other validators to follow this structure?